src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue

```diff
@@ -0,0 +1,233 @@
+		path: ["Site Layout", "Twitch Features"],
+		label: "Hide Bits Buttons",
+		hint: "If checked, the 'Bits' related buttons will be hidden",
+		defaultValue: false,
+	}),
+	declareConfig("hide.subscribe_button", "TOGGLE", {
+		path: ["Site Layout", "Twitch Features"],
+		label: "Hide Subscribe Button",
+		hint: "If checked, the 'Subscribe' button will be hidden (includes 'gift a sub' button)",
+		defaultValue: false,
+	}),
+	declareConfig("hide.prime_offers", "TOGGLE", {
+		path: ["Site Layout", "Twitch Features"],
+		label: "Hide Prime Offers",
+		hint: "If checked, the 'Prime Offers' button on the top bar will be hidden",
+		defaultValue: false,
+	}),
+	declareConfig("hide.unfollow_button", "TOGGLE", {
+		path: ["Site Layout", "Twitch Features"],
+		label: "Hide Unfollow Button",
+		hint: "If checked, the 'Unfollow' button will be hidden (under stream and in theater mode)",
+		defaultValue: false,
+	}),
+	declareConfig("hide.live_notification_button", "TOGGLE", {
+		path: ["Site Layout", "Twitch Features"],
+		label: "Hide Live Notification Button",
+		hint: "If checked, the 'Turn notifications off/on' button under the stream will be hidden",
+		defaultValue: false,
+	}),
+	// Side bar elements
+	declareConfig("hide.recommended_channels", "TOGGLE", {
+		path: ["Site Layout", "Side Bar"],
+		label: "Hide Recommended Channels",
+		hint: "If checked, the 'recommended channels' section of the side bar will be hidden",
+		defaultValue: false,
+	}),
+	declareConfig("hide.viewers_also_watch", "TOGGLE", {
+		path: ["Site Layout", "Side Bar"],
+		label: "Hide Viewers Also Watch",
+		hint: "If checked, the 'viewers also watch' section of the side bar will be hidden",
+		defaultValue: false,
+	}),
+	// Player elements
+	declareConfig("hide.top_bar_of_stream", "TOGGLE", {
+		path: ["Site Layout", "Video Player"],
+		label: "Hide Top Bar of Stream",
+		hint: "If checked, the top bar of the stream which shows on hover will be hidden",
+		defaultValue: false,
+	}),
+	declareConfig("hide.player_controls", "TOGGLE", {
+		path: ["Site Layout", "Video Player"],
+		label: "Hide Player Controls",
+		hint: "If checked, the controls shown at the bottom of a stream on hover will be hidden",
+		defaultValue: false,
+	}),
+];
+</script>
+
+<style lang="scss">
+.seventv-hide-leaderboard {
+	div[data-test-selector="channel-leaderboard-container"] {
+		display: none !important;
+	}
+}
+
+.seventv-hide-buttons-below-chatbox {
+	div[data-test-selector="chat-input-buttons-container"] {
+		display: none !important;
+	}
+}
+
+.seventv-hide-stream-chat-bar {
+	button[data-a-target="right-column__toggle-collapse-btn"],
+	div[class$="stream-chat-header"] {
+		display: none !important;
+	}
+}
+
+.seventv-hide-react-buttons {
+	div[class$="theatre-social-panel"]
+		button[class="ScCoreButton-sc-ocjdkq-0 ScCoreButtonSecondary-sc-ocjdkq-2 ibtYyW bTKXKk"],
+	div[data-target="channel-header-right"]
+		button[class="ScCoreButton-sc-ocjdkq-0 ScCoreButtonText-sc-ocjdkq-3 ibtYyW jYfhUy"] {
```

Comment:
The classes here are generated and will change anytime Twitch deploys a new version of their site.